dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/sme/SmeData.kt

```diff
@@ -0,0 +1,86 @@
+package org.dataland.datalandbackend.model.sme
+
+import org.dataland.datalandbackend.annotations.DataType
+import org.dataland.datalandbackend.model.enums.commons.YesNoNa
+import org.dataland.datalandbackend.model.enums.sme.CompanyAgeBracket
+import org.dataland.datalandbackend.model.enums.sme.EnergyEfficiencyBracket
+import org.dataland.datalandbackend.model.enums.sme.EnergyProductionBracket
+import org.dataland.datalandbackend.model.enums.sme.HeatSource
+import org.dataland.datalandbackend.model.enums.sme.Industry
+import java.math.BigDecimal
+
+/**
+ * --- API model ---
+ * Fields of the questionnaire for the SME framework
+ */
+@DataType("sme")
+data class SmeData(
+    val industry: Industry?,
+
+    val financialYear: Long?,
```

Comment:
Don't we use BigDecimal for any field with Number as expected value? Otherwise also in the SfdrData there could be BigDecimals be replaced with some integer type. None of your business, just a note.

Reply:
Wie besprochen geändert.